docs/Configuration.md

```diff
@@ -1,40 +1,220 @@
 # Configuration
 
+* [Extensions](#extensions)
 * [Receivers](#receivers)
 * [Processors](#processors)
 * [Exporters](#exporters)
 
+## Extensions
+
+In order to send data to [Sumo Logic][sumologic_webpage] one needs to configure
+the [sumologicextension][sumologicextension] with credentials and define it
+(the extension) in the same service as [sumologicexporter][sumologicexporter]
+is defined so that it's used as auth extension.
+
+The basic example to collect CPU load metrics using [Host Metrics Receiver][hostmetricsreceiver]
+and send them to Sumo Logic has following form:
+
+```yaml
+extensions:
+  sumologic:
+    access_id: <my_access_id>
+    access_key: <my_access_key>
+    collector_name: <my_collector_name>
+
+# Define your receivers...
+receivers:
+  hostmetrics:
+    collection_interval: 30s
+    scrapers:
+      load:
+
+# Define your processors...
+processors:
+
+exporters:
+  sumologic:
+    auth:
+      authenticator: sumologic
```

Comment:
Can we add example with `sumologic/1`?
`sumologic` is default auth in exporter now

Reply:
Separate example explaining `sumologic/1`

Reply:
After looking in code we don't recognize multiple extensions, so no need to document it
@pmalek-sumo is that correct statement?

Reply:
Yup, for now we only take the first one found. https://github.com/SumoLogic/sumologic-otel-collector/blob/d4157051abcfb6a82baaf587d8a813afe3e04a45/pkg/exporter/sumologicexporter/exporter.go#L379-L386

We should change that to account for more extensions being able to be defined, that would (as far as I can tell for now) require exposing Component ID from `config.ExtensionSettings`

Reply:
@sumo-drosiek @pmalek-sumo I propose to change documentation when we add support for multiple authenticators in the config.